components/builder-web/app/origin/origin-page/origin-packages-tab/origin-packages-tab.component.html

```diff
@@ -8,32 +8,20 @@
         (saved)="saved($event)">
       </hab-project-settings>
     </section>
+
     <section *ngIf="!selectingPlan">
-      <ul class="nav-list" *ngIf="projectsExist">
+      <ol class="nav-list">
         <li class="heading">
-          <h4>Connected Plans</h4>
-        </li>
-        <li class="item" *ngFor="let proj of projects">
-          <a [routerLink]="['/pkgs', origin, proj, 'latest']">
-            <span class="column name">{{ proj }}</span>
-          </a>
-          <hab-icon symbol="chevron-right"></hab-icon>
-        </li>
-      </ul>
-      <ul class="nav-list">
-        <li class="heading">
-          <h4>Packages</h4>
-        </li>
-        <li class="none" *ngIf="noPackages">
-          <span>No packages found.</span>
+          <h4>Package Name</h4>
         </li>
         <li class="item" *ngFor="let pkg of packages">
           <a [routerLink]="['/pkgs', pkg.origin, pkg.name, 'latest']">
             <span class="column name">{{ pkg.name }}</span>
+            <hab-icon symbol="chevron-right" class="chevron-right"></hab-icon>
           </a>
-          <hab-icon symbol="chevron-right"></hab-icon>
         </li>
-      </ul>
+      </ol>
```

Comment:
Turns out this was the only page using Tables, so I converted it over to a list element and then all the styling for it was set globally 🙌